powerauth-webflow/src/main/java/io/getlime/security/powerauth/app/webflow/configuration/SecurityConfiguration.java

```diff
@@ -38,7 +38,7 @@ protected void configure(HttpSecurity http) throws Exception {
                 .httpBasic().disable()
                 .csrf().ignoringAntMatchers("/api/auth/token/app/**", "/pa/**").and()
                 .antMatcher("/**").authorizeRequests()
-                .antMatchers("/", "/authenticate", "/authenticate/**", "/oauth/error", "/api/**", "/pa/**", "/resources/**", "/ext-resources/**", "/websocket/**").permitAll()
+                .antMatchers("/", "/authenticate", "/authenticate/**", "/oauth/error", "/api/**", "/pa/**", "/resources/**", "/ext-resources/**", "/websocket/**", "/swagger-ui.htm", "/v2/api-docs").permitAll()
```

Comment:
The Swagger UI URL in SecurityConfiguration is incorrect. The suffix should be .html, not .htm.